Header header
geometry_msgs/PolygonStamped target_plane
# parameters, 0 means 
float64 max_distance
float64 eps_angle
